packages/preset-uno/package.json

```diff
@@ -22,30 +22,26 @@
   "sideEffects": false,
   "exports": {
     ".": {
-      "types": "./dist/index.d.ts",
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.cjs"
+      "types": "./dist/index.d.mts",
+      "import": "./dist/index.mjs"

```

Comment:
iirc this should be "default" instead "import"

Reply:
root types should use `.d.ts`, `.d.mts` will use `.js` extension and that's wrong in node10

Reply:
I'm very busy at work, I'll check the changes this afternoon ( /cc @kwaa )